sklearn/ensemble/bagging.py

```diff
@@ -33,21 +33,55 @@
 MAX_INT = np.iinfo(np.int32).max
 
 
+def _generate_mask_from_indices(indices, mask_length, inverse=False):
```

Comment:
Ok, I'll get rid of the `inverse` argument. Any idea where in `utils` this would go?